MidCamp/midcamp
# 2020 Schedule

## Description

Adds 2020 schedule

## To test

Visit https://nginx-midcamp-org-feature-schedule-2020.us.amazee.io/ and observe the links appearing under the "Schedule" parent menu item.
- Trainings appear at https://nginx-midcamp-org-feature-schedule-2020.us.amazee.io/2020/schedule/wednesday
- Thursday sessions appear at https://nginx-midcamp-org-feature-schedule-2020.us.amazee.io/2020/schedule/thursday
- Friday sessions appear at https://nginx-midcamp-org-feature-schedule-2020.us.amazee.io/2020/schedule/friday
- Contribution day placeholder page at https://nginx-midcamp-org-feature-schedule-2020.us.amazee.io/2020/contribution-day
